documentation_crawl4ai/adaptive-strategies.md

```
@@ -0,0 +1,520 @@
+where score(t, K) = doc_coverage(t) × (1 + freq_boost(t))
+Copy
+```
+
+#### Components
+  * **Document Coverage** : Percentage of documents containing the term
+  * **Frequency Boost** : Logarithmic bonus for term frequency
+  * **Query Decomposition** : Handles multi-word queries intelligently
+
+
+#### Tuning Coverage
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)